### ISSUE TITLE
using filemanager in api

### ISSUE THREAD

**Issue by fmmajd** [UniSharp/laravel-filemanager#465]
hi

I'm using Laravel mainly as a backend service. the project's frontend is written in angular 2 and communicates through API. I wanna use the file manager in ckeditor in my frontend,(admin panel mainly). but I'm stuck on what to do for it. the default web and auth middlewares will fire the error 'route[login]' not found(which is logical. I haven't used them). if I use only the web middleware, the error 'trying to get property of non-object' comes. 
For our own use, we have a custom middleware called custom_api, and when I use that, I get an error "Class custom_api does not exist".

 so basically, how can I totally disable the authentication, and write my own?

**Comment by g0110280** [comment]
For the error you got, maybe you should check if you set up route middlewares in `app/Http/Kernel.php`. And by the way, the api of file manager contains html. It is also our milestone in v2.0 to leave only json data in the api.

**Comment by fmmajd** [comment]
i could ALMOST get it to work. for people reading it, something that worked for me was to disable default routes, copy all the routes in my own route file and do not use any middleware on the routes. then implement your own auth system.

**Comment by g0110280** [comment]
Done in `v2.0.0-alpha`, `ItemController` now returns json response.
